### PR TITLE
Changes to hackclub.com/slack post discussion with @zrl

### PR DESCRIPTION
**Pull request by jaspermayone**

### CODE REVIEW

**Comment by jaspermayone** [comment]
@zachlatta open and ready for review plz.

File: components/slack/join-form.js [hunk 1]
```diff
@@ -93,10 +93,10 @@ const JoinForm = ({ sx = {}, router }) => {
           </Label>
         </Grid>
         <Label>
-          Why do you want to join the Hack Club Slack?
+          How did you hear about us/the slack? What are you most looking forward to?
```
Comment (sampoder) [inline components/slack/join-form.js]:
```suggestion
          How did you hear about us/the Slack? What are you most looking forward to?
```

**Comment by jaspermayone** [comment]
ty sam!

**Comment by jaspermayone** [comment]
Will test once deploys!

**Comment by jaspermayone** [comment]
working!
CleanShot 2024-05-20 at 12 01 47@2x

**Pull request merged**